### DISCUSSION TITLE
Having problems renewing my educational access

### DISCUSSION THREAD

**Discussion by trdunsworth** [community/community#81374] [GitHub Education]
### Select Topic Area

Product Feedback

### Body

When I first signed up, there were no problems. Now I have to have billing on file and because I'm a remote student, as is most of the student body, I can't be verified. This is just too many hoops to put up with for the little benefit I receive. Can someone in administration help address this nightmare.

**Comment by faridnec** [comment]
Hey, I got the same issue. I decided to reach them through their Help Center: [https://support.github.com/](https://support.github.com/). No answer yet tho

**Reply by trdunsworth** [reply]
I may try that as well. Thanks for the feedback. Let me know if you end up getting a response.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @trdunsworth , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
